read-fonts/src/tables/truetype.rs

```diff
@@ -0,0 +1,3 @@
+//! TrueType (glyf) common code.
```

Comment:
Is this a table? I might make it a submodule of `glyf` if that is more logical, we do this for a few other tables.

Reply:
I waffled on this because technically bytecode applies to glyf, fpgm and prep so it is more of a common thing. But I think putting it under glyf makes sense and removes the otherwise empty truetype module.